pkg/gce-pd-csi-driver/node.go

```diff
@@ -347,11 +345,13 @@ func (ns *GCENodeServer) GetVolumeLimits() (int64, error) {
 	if err != nil {
 		return volumeLimits, err
 	}
+	var str = string(body[:])
+	*/
 
 	// Response format: "projects/[NUMERIC_PROJECT_ID]/machineTypes/[MACHINE_TYPE]"
-	var str = string(body[:])
-	splits := strings.Split(str, "/")
-	var machineType = splits[len(splits)-1]
+	fullMachineType := ns.MetadataService.GetMachineType()
```

Comment:
Can you also do this parsing inside metadata.go?

Reply:
Sure, I'm working on it.

Reply:
I have updated the code, please help to review it